test/e2e/scaffold/ingress.go

```diff
@@ -217,7 +217,7 @@ spec:
       initContainers:
       - name: wait-apisix-admin
         image: localhost:5000/busybox:1.28
-        command: ['sh', '-c', "until nc -z apisix-service-e2e-test.%s.svc.cluster.local 9080 ; do echo waiting for apisix-admin; sleep 2; done;"]
+        command: ['sh', '-c', "until nc -z apisix-service-e2e-test.%s.svc.cluster.local 9180 ; do echo waiting for apisix-admin; sleep 2; done;"]
```

Comment:
Here we add another template `%s`, but it seems you don't change the place we render it.

Reply:
https://github.com/apache/apisix-ingress-controller/pull/660/files#diff-37e06787f8794acb42268c2dde87b4907b54af48e47fdb3b4e2648f97c9fc65aR350 the %s is replaced here.

Reply:
OK.